libraries/AP_HAL_Linux/Scheduler.cpp

```diff
@@ -60,6 +60,8 @@ extern const AP_HAL::HAL& hal;
         .rate = APM_LINUX_##UPPER_NAME_##_RATE,                 \
     }
 
+#define DEBUG_STACK 0
+
```

Comment:
I am not a fan of having DEBUG defines inside the code. Could you just use a #if defined and pass the param from the build system if needed? Or activated by a param ?